### DISCUSSION TITLE
Disable model_validate

### DISCUSSION THREAD

**Discussion by ramkumar-ashok** [mirumee/ariadne#1145] [Q&A]
Hi all, I was able to generate functions, but I want to remove model_validate in each of the function. 
How do I do that? Are there any configuration that allows it?

**Comment by rafalp** [comment]
Hello,

To remove `model_validate` calls from generated client, you should [implement a plugin](https://github.com/mirumee/ariadne-codegen/blob/main/PLUGINS.md) that uses the [`generate_client_method`](https://github.com/mirumee/ariadne-codegen/blob/main/PLUGINS.md#generate_client_method) to modify returned AST to instantiate return types directly instead of calling their `model_validate` methods.
